parcels/field.py

```diff
@@ -267,9 +267,11 @@ def from_netcdf(cls, filenames, variable, dimensions, indices={},
 
         if not isinstance(filenames, Iterable) or isinstance(filenames, str):
             filenames = [filenames]
+        indices = indices.copy()
```

Comment:
why do we need to copy? If this is necessary, perhaps add a short comment on the rationale